snippets/icon-accordion.liquid

```diff
@@ -1,5 +1,14 @@
 {%- if icon != 'none' -%}
-  <svg class="icon icon-accordion color-foreground-{{ settings.accent_icons }}" aria-hidden="true" focusable="false" role="presentation" xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20">
+  <svg
+    class="icon icon-accordion color-foreground-{{ settings.accent_icons }}"
+    aria-hidden="true"
+    focusable="false"
+    role="presentation"
+    xmlns="http://www.w3.org/2000/svg"
+    width="20"
+    height="20"
+    viewBox="0 0 20 20"
+  >
```

Comment:
I wonder if this should also apply to SVG elements in liquid files like this for icons. 🤔  Definitely easier to read but I feel like it isn't necessary.

Reply:
https://github.com/Shopify/prettier-plugin-liquid/discussions/15